features/step_definitions/basic_steps.rb

```diff
@@ -94,6 +63,17 @@
   switch_to_window(windows[index.to_i - 1])
 end
 
+When("I click {string} on rating") do |value|
+  within('#rating') do
+    click_on value
+  end
+  sleep 2
+end
+
+Then("stop") do
```

Comment:
Was this function used? If not, remove it.

Reply:
We have kept this for troubleshooting further in project then we dont have to keep adding it.

Reply:
It’s a debugger. Please leave it in the code. Used in development